src/test/osdc/object_cacher_stress.cc

```diff
@@ -47,7 +47,7 @@ class C_Count : public Context {
     : m_op(op), m_outstanding(outstanding) {}
   void finish(int r) override {
     m_op->done++;
-    assert(m_outstanding > 0);
+    assert(*m_outstanding > 0);
     *m_outstanding--;
```

Comment:
Hi, @joscollin, I don't see any other use for it.  On line 71, the pointee variable "outstanding_reads" is declared, and then on 104 it's incremented, only to be decremented here when finish() is called. It looks like the assertion is checking that there's at least one outstanding read, but what I /don't/ see is a check to see if the final value is again zero after all operations are complete.

However, I didn't write the original code, so I'm also second-guessing the author. :-\